locales/en.default.schema.json

```diff
@@ -1912,6 +1912,54 @@
               "label": "Enable quick add button"
             }
           }
+        },
+        "icon_with_text": {
+          "name": "Icon with text",
+          "settings": {
+            "layout": {
+              "label": "Layout",
+              "options__1": {
+                "label": "Horizontal"
+              },
+              "options__2": {
+                "label": "Vertical"
+              }
+            },
+            "content": {
+              "label": "CONTENT",
```

Comment:
I don't think this needs to be all caps, does it? I feel like that's a style that's up to the editor.

Reply:
I believe the editor adds the all caps styling automatically to content labels. Meaning I'd expect you could edit this and the result in the editor will look the same. In general, source text content should rarely enforce any styling decisions, like all caps. That should usually be left to the CSS/UI to apply as needed.

Compare the rendered result of this
<img src="https://screenshot.click/23-25-u7hep-x1ap6.png" width="300"/>

To this
<img src="https://screenshot.click/23-25-ht9d1-xzf3w.png" width="300"/>